doc/installation.xml

```diff
@@ -145,8 +145,86 @@ $HOME/.nix-profile/etc/profile.d/hm-session-vars.sh
   <title>NixOS module</title>
 
   <para>
-   To be done.
+   Home Manager provides a NixOS module that allows you to prepare user
+   environments directly from the system configuration file, which often is
+   more convenient than using the <command>home-manager</command> tool. It also
+   opens up additional possibilities, for example, to automatically configure
+   user environments in NixOS declarative containers or on systems deployed
+   through NixOps.
+  </para>
+
+  <para>
+   To make the NixOS module available for you you must <option>import</option>
+   it into your system configuration. This is most conveniently done by adding
+   a Home Manager channel, for example
+  </para>
+
+<screen>
+<prompt>#</prompt> <userinput>nix-channel --add https://github.com/rycee/home-manager/archive/master.tar.gz home-manager</userinput>
+<prompt>#</prompt> <userinput>nix-channel --update</userinput>
+</screen>
+
+  <para>
+   if you are following Nixpkgs master or an unstable channel and
+  </para>
+
+<screen>
+<prompt>#</prompt> <userinput>nix-channel --add https://github.com/rycee/home-manager/archive/release-18.09.tar.gz home-manager</userinput>
+<prompt>#</prompt> <userinput>nix-channel --update</userinput>
```

Comment:
should we also mention that you can simply import `"${homeManager}/nixos` and have `homeManager` set to fetchTarball?

Reply:
I've throught about this and I think I'd prefer to avoid adding such additional descriptions for now to keep things simple.

If I can think of a good way to include this without complicating things then I'll write something up 🙂